lib/ably/rest/client.rb

```diff
@@ -140,18 +145,19 @@ def initialize(options)
           end
         end
 
-        @realtime_client  = options.delete(:realtime_client)
-        @tls              = options.delete(:tls) == false ? false : true
-        @environment      = options.delete(:environment) # nil is production
-        @environment      = nil if [:production, 'production'].include?(@environment)
-        @protocol         = options.delete(:protocol) || :msgpack
-        @debug_http       = options.delete(:debug_http)
-        @log_level        = options.delete(:log_level) || ::Logger::WARN
-        @custom_logger    = options.delete(:logger)
-        @custom_host      = options.delete(:rest_host)
-        @custom_port      = options.delete(:port)
-        @custom_tls_port  = options.delete(:tls_port)
-        @add_request_ids  = options.delete(:add_request_ids)
+        @realtime_client     = options.delete(:realtime_client)
+        @tls                 = options.delete(:tls) == false ? false : true
```

Comment:
`!(options.delete(:tls) == false)`

Reply:
Sure, we could do lots of Ruby golf throughout, but I prefer to have the code as readable as possible. So it's a style preference of course, but I think a ternary expression is easier to comprehend.  BTW. This code was not changed, it was just indented.